test/integration/auth/node_test.go

```diff
@@ -107,7 +107,7 @@ func TestNodeAuthorizer(t *testing.T) {
 	masterConfig.GenericConfig.AdmissionControl = nodeRestrictionAdmission
 
 	// enable testing volume attachments
-	masterConfig.ExtraConfig.APIResourceConfigSource.(*serverstorage.ResourceConfig).EnableVersions(storagev1alpha1.SchemeGroupVersion)
+	masterConfig.ExtraConfig.APIResourceConfigSource.(*serverstorage.ResourceConfig).EnableVersions(storagev1beta1.SchemeGroupVersion)
```

Comment:
Should be able to remove this line if the beta version is enabled by default

Reply:
done